devtools/test_dashboard_v2/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Plotly.js Devtools</title>
+  <style type="text/css">
+    html, body{
+      margin: 0;
+      padding: 0;
+      font-family: helvetica, arial, sans-serif;
+    }
+    header{
+      height: 40px;
+      border-bottom: 1px solid #4983EC;
+    }
+    header img{
+      height: 100%;
+      display: inline-block;
+      vertical-align: top;
+    }
+    header input{
+      float: right;
+      background-color: none;
+      border: none;
+      border-bottom: 1px solid #ddd;
+      padding: 5px;
+      outline: none;
+      margin-top: 6px;
+      margin-right: 10px;
+    }
+    #reload-time{
+      color: #bbb;
+      font-size: 10px;
+      line-height: 40px;
+    }
+    #mocks-list{
+      position: absolute;
+      right: 0px;
+      top: 40px;
+      z-index: 1000;
+      background-color: #fff;
+      border-left: 1px solid #4983EC;
+      border-bottom: 1px solid #4983EC;
+      border-top: 1px solid #4983EC;
+    }
+    .search-result{
+      display: block;
+      margin-bottom: 2px;
+      padding: 5px 10px;
+      text-align: right;
+      font-size: 12px;
+    }
+    .search-result:hover{
+      color: #fff;
+      background-color: #4983EC;
+    }
+    .dashboard-plot{
+      margin-bottom: 30px;
+    }
+  </style>
+</head>
+<body>
+  <header>
+    <img src="http://images.plot.ly/logo/plotlyjs-logo@2x.png" onClick="Tabs.reload();">
+    <span id="reload-time"></span>
+
+    <input id="mocks-search" type="text" placeholder="mocks search"></input>
+  </header>
+
+  <section id="mocks-list"></section>
+  <div id="plots"></div>
+  <div id="snapshot"></div>
+
+  <script id="source" type="text/javascript" src="../../build/plotly.js"></script>
+  <script type="text/javascript" src="./fuse.min.js"></script>
```

Comment:
I'd vote for having a  `require('fuse')` in `devtools.js` and then browserify `devtools.js` on server boot.